keyboards/jones/v.0.3/matrix.c

```diff
@@ -0,0 +1,234 @@
+/*
```

Comment:
Also, could this be converted to the "lite" custom matrix?

Reply:
Also, converted to "lite" version.